docs/assemblies/tls.adoc

```diff
@@ -107,10 +107,12 @@ to the SAN.
 
 ==== issuer
 
-This attribute corresponds to the certmanager issuer that will be used to issue the certificate.
-If the issuers attribute is not set (as in the configuarion above for service1), certificates
-will be issued using the default root CA for internal TLS as defined in lib-common.
-(currently set to "osp-rootca-issuer-internal").
+This attribute corresponds to a label for the certmanager issuer that will be used to issue the
+certificate.  Note that this can be different from the name of the issuer.  It is expected that there
+is only one issuer with the specified label.  If ore than one issuer has the label, an error will be
+generated.  If the issuers attribute is not set (as in the  configuration above for service1), certificates
+will be issued using the default root CA for internal  TLS as defined in lib-common. (currently set to
+"osp-rootca-issuer-internal" for the rootca-internal issuer).
```

Comment:
```suggestion
This attribute corresponds to the label for the certmanager issuer that is used to issue the
certificate. The label can be different from the name of the issuer.  There can be only one issuer with the specified label.  If more than one issuer has the label, an error is generated. If the issuers attribute is not set, as in the  configuration for `service1`, the certificates are issued with the default root CA for internal  TLS as defined in `lib-common`, which is set to "osp-rootca-issuer-internal" for the `rootca-internal` issuer.
```
Hey, some suggestions for minimalism/clarity, and to align with style guide:

- Avoid future tense.
- Avoid parenthesis in text.

Reply:
thanks!  will change as provided